systempony

```diff
@@ -110,16 +129,19 @@ function ponyget_Packages()
     elif which pacman &>/dev/null
     then
         pacman -Q | wc -l
+    elif which apk &>/dev/null
+    then
+	apk list | wc -l
     fi
 }
 
 function ponyget_Disk()
 {
     local dfoption=$FSTYPE
-    [ "$dfoption" ] && dfoption="-t $FSTYPE"
-    local diskusage=$(df -lh $dfoption --total | tail -n 1 | sed -r "s/ +/\t/g" )
-    local used=$(echo "$diskusage" | cut -f 3)
-    local total=$(echo "$diskusage" | cut -f 2)
+    [ "$dfoption" ] && dfoption="grep $FSTYPE" || dfoption="cat"
+    local diskusage=$(df -h -T | $dfoption | tail -n 1 | sed -r "s/ +/\t/g" )
```

Comment:
Removing `--total` breaks the expected behaviour when you have multiple matching filesystems mounted

Reply:
Removing the `-l` switch also changes the way it works considerably. Feel free to improve on this if you want. I just submitted this as a pull request to bring it to your attention. This works for what I'm doing for now.